### ISSUE TITLE
binding: requires a date attribute of some kind @contemporary = 'true'

### ISSUE THREAD

**Issue by mjhawkins** [cambridge-collection/cdcp-schemata#22]
Is this a rule we want to keep in the new universal schema or an Oxford-specific implementation.

**Comment by Dr-James-Freeman** [comment]
I think @contemporary="true" is problematic in a way that assigning a @notAfter / @notBefore date range is not. Contemporary with what? The date of creation of the manuscript? Or meaning "contemporary with the present era" (aka "modern")? There is an ambiguity there. 

A cataloguer might reasonably assume that "contemporary with the creation of the manuscript" is what is intended, but that too risks implying something about medieval book production that we cannot prove, except in very rare instances: i.e. that the binding of manuscripts took place at or shortly after their production had been 'completed' (whatever that means in a culture of bespoke manuscript production).

**Comment by ElizabethGow** [comment]
I'm generally in favour of providing dating information wherever possible, but think this should be recommended rather than mandatory. When dating information is given, this should be the dating information (notBefore/notAfter/when) AND the contemporary attribute (if a cataloguer can give a date range, they can state if contemporary= "true", "false" or "unknown", and vice versa). This could support a useful browse function, but I don't think should be used to generate displayed text.

**Comment by Dr-James-Freeman** [comment]
I agree that it shouldn't be used to generate displayed text - the cataloguer can include a prose description of the likely date of the binding, which could be as specific as "16th century" (or more specific) or as general as "medieval" / "post-medieval". I do think that dating of some sort ought to be mandatory, however, since the omission of this data would mean that the manuscript is invisible if someone is filtering manuscripts by date of binding (unless there were also an option for "undated" - for those who like a challenge, perhaps, or want to do their due diligence and leave no stone unturned).

**Comment by Dr-James-Freeman** [comment]
Also: not an Oxford-specific implementation: we have been using this at the UL.

**Comment by mjhawkins** [comment]
Leave as it is. Keep as reminder in GS.

The issue of the contents will still need to be discussed.
